demux/demux_playlist.c

```diff
@@ -277,6 +277,8 @@ static int parse_m3u(struct pl_parser *p)
                 title = bstrto0(NULL, btitle);
             }
         } else if (bstr_startswith0(line_dup, "#EXT-X-")) {
+            // Note that this only used for user advice. Actual HLS demuxing must
```

Comment:
> Note that this only used for user advice.

What does this mean exactly?

Reply:
I see, it wasn't clear to me. 

Also, I've tested and yes HLS has to be demuxed by lavf to get playlist reload. mpv's playlist demuxer would just play chunks from single playlist and never loads new chunks.